Makefile

```diff
@@ -11,6 +11,7 @@ docker:
 install: 
 	pip install -r requirements.txt
 	pip3 install target/dist/stanford-wdl-kit-$(VERSION)/dist/stanford-wdl-kit-$(VERSION).tar.gz
+	pip install pysftp
```

Comment:
this needs to go into requirements.txt so that the Dockerfile will pick it up also